File: 백준/Bronze/1977. 완전제곱수/완전제곱수.py
M = int(input())
N = int(input())

cnt = 0
sum = 0
min = -1

for i in range(M, N + 1):
    for j in range(1, i + 1):
        if pow(j, 2) == i:
            cnt += 1
            sum += i
            if cnt == 1:
                min = i
                
if min == -1:
    print(-1)

else:
    print(sum)
    print(min)

File: 백준/Bronze/1977. 완전제곱수/완전제곱수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int start = Integer.parseInt(br.readLine());
        int end = Integer.parseInt(br.readLine());

        int res = 0, min = Integer.MAX_VALUE;
        for (int i = start; i <= end; i++) {
            for (int j = 0; j <= i; j++) {
                if (Math.pow(j, 2) == i) {
                    res += i;
                    min = Math.min(min, res);
                }
            }
        }

        if (res == 0) System.out.println(-1);
        else System.out.println(res + "\n" + min);
    }
}